integrations/terraform/go.mod

```diff
@@ -18,17 +18,21 @@ require (
 	github.com/olekukonko/tablewriter v0.0.5
 	github.com/sirupsen/logrus v1.9.3
 	github.com/stretchr/testify v1.9.0
-	golang.org/x/exp v0.0.0-20240325151524-a685a6edb6d8
+	golang.org/x/exp v0.0.0-20240325151524-a685a6edb6d8 // indirect
```

Comment:
Can we move this to the `require` block below? So that all `indirect` deps are grouped